enclave_apps/Cargo.lock

```diff
@@ -910,7 +916,7 @@ version = "0.36.9"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "fd5c6ff11fecd55b40746d1995a02f2eb375bf8c00d192d521ee09f42bef37bc"
 dependencies = [
- "bitflags",
```

Comment:
Is it ok that there are still a few of these "bitflags 1.3.2"?

Reply:
Yes. Tonic still indirectly depends on version 1.3.2, but Tonic is not used in the Restricted Kernel or Stage 0 where we are trying to manage the dependencies tightly.